core/src/tangram.h

```diff
@@ -166,6 +166,9 @@ void runOnMainLoop(std::function<void()> _task);
 // Run this task asynchronously to Tangram's main update loop.
 void runAsyncTask(std::function<void()> _task);
 
+// Capture a snapshot of the current frame
+std::vector<unsigned int> captureSnapshot();
```

Comment:
We can do better with this interface. The big issues are 1) the width, height, and pixel size of the captured image are not obvious and 2) `std::vector` is a pretty restrictive container, you can't get the data out without copying. 

1) is easily resolved by taking in-out integer arguments to populate with the image width, height, and pixel size.

2) is normally solved by taking a pointer argument to a buffer, but the trouble is then ensuring a large enough buffer.